requirements/docs.txt

```diff
@@ -2,15 +2,15 @@
 # This file is autogenerated by pip-compile
 # To update, run:
 #
-#    pip-compile requirements/docs.in
+#    pip-compile /home/jon/projects/h/requirements/docs.in
```

Comment:
Full path here?

Reply:
Oh that's unfortunate... yeah this is going to be due to specifying the files absolutely I'd guess...

Reply:
Wait, why only this one file!?